### PR TITLE
moved to global monetize

### PR DESCRIPTION
**Pull request by delianides**

### CODE REVIEW

**Comment by jbaxleyiii** [comment]
**[Storybook Hub](https://hub.getstorybook.io/)** is building storybooks for every commit in this PR.

**Latest Storybook: https://315487f9-0ace-4811-9d04-682750b93304.sbook.io/**
<sub>( Visit [here](https://hub.getstorybook.io/app/github.com/NewSpring/Holtzman/branch/refactor-give-review-2513/) for more info )</sub>

> _This comment is automatically added by Storybook Hub_

**Comment by delianides** [comment]
Don't merge this just yet.

File: imports/pages/give/review/index.js [hunk 1]
```diff
@@ -104,15 +84,17 @@ export class Template extends Component {
           additionalMessage={
             "Please click 'Done' in the top left of your screen to get back to the app"
           }
+          goToStepOne={() => {}}
```
Comment (delianides) [inline imports/pages/give/review/index.js]: @johnthepink @jbaxleyiii these props are required by flow and making them optional causes a series of cascading flow failures. Is there a better way to satisfy this prop?

Comment (jbaxleyiii) [inline imports/pages/give/review/index.js]: @delianides why are we not just passing the `goToStepOne` function?

Comment (jbaxleyiii) [inline imports/pages/give/review/index.js]: oh! maybe because this is for the ios review process

Comment (jbaxleyiii) [inline imports/pages/give/review/index.js]: in which case a noop is a good call

**Comment by delianides** [comment]
@jbaxleyiii if this part is for iOS and you're ok with the noop then this is good to go from me.

**Pull request merged**